src/inference/Inference.jl

```diff
@@ -865,8 +870,8 @@ function dot_observe(
     vi::VarInfo,
 )
     increment_num_produce!(vi)
-    Turing.DEBUG && @debug "dist = $dist"
-    Turing.DEBUG && @debug "value = $value"
+    Turing.DEBUG && _debug("dist = $dist")
```

Comment:
IIRC we need `_debug` due to some Zygote issues?

Reply:
Yes